document/core/appendix/changes.rst

```diff
@@ -138,6 +138,24 @@ Added vector type and instructions that manipulate multiple numeric values in pa
 * New injection/projection :ref:`vector instructions <syntax-instr-vec>`: :math:`\K{i}\!N\!\K{x}\!M\!\K{.splat}`, :math:`\K{f}\!N\!\K{x}\!M\!\K{.splat}`, :math:`\K{i}\!N\!\K{x}\!M\!\K{.bitmask}`
 
 
+.. index:: instructions, exception, tag type, tag, handler
+
+Exception Handling
+..................
+
+Added tag type, instructions that throw exceptions of a tag type, and instructions that handle exceptions. [#proposal-eh]_
+
+* New :ref:`tag type <syntax-tagtype>`: :math:`[t^\ast]\to[]`-
```

Comment:
I'd replace this with tag definitions, imports, and exports. Tag types are more of an auxiliary notion to support these constructs.